recipes/rmm/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "rmm" %}
+{% set version = "21.12.00" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rapidsai/rmm/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 71802422a04f2e0a828f7b7ec77e5cd1d24361d6dc5bf60a534659e4c2e08a44
+  folder: rmm
+
+build:
+  number: 0
+  skip: true  # [(not linux) or (cuda_compiler_version ==  "None") or (aarch64) or (py < 37) or (py >= 39)]
+
+  script_env:
+    - INSTALL_PREFIX=${PREFIX}
```

Comment:
Note: You removed the `CUDA_HOME` env var and it would not work when actually compiling for ppc64le in the feedstock.

Reply:
Added back.